python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -54,14 +56,16 @@ class _Job:
 
     Parameters
     ----------
-    qdata : `~lsst.pipe.base.QuantumIterData`
+    qdata : `~lsst.pipe.base.QuantumNode`
         Quantum and some associated information.
     """
```

Comment:
CAn you add docstring for two new arguments.

Reply:
They were not needed any more and were removed instead